### ISSUE TITLE
FlatFeed real-time notification not working

### ISSUE THREAD

**Issue by brunoluiz** [GetStream/react-activity-feed#90]
We are using the following code for one of our flat feeds. It doesn't seem to be receiving data from our web socket, as it never renders the `Notifier` component when we push new activities to the stream.

```js
import 'react-activity-feed/dist/index.es.css';
import { StreamApp, FlatFeed } from 'react-activity-feed';
import React from 'react';
import ReactDOM from 'react-dom';

const API_KEY = "...";
const APP_ID = "...";
const TOKEN = "...";

const App = () => <StreamApp apiKey={API_KEY} appId={APP_ID} token={TOKEN} >
  <FlatFeed 
    notify
    feedGroup="STREAM_XYZ"
    Activity={(props) => <pre>{JSON.stringify(props)}</pre>}
  />
</StreamApp>;

ReactDOM.render(<App />, document.getElementById('root'));
```

Dependencies:

```
  ...
  "dependencies": {
    "react": "^16.12.0",
    "react-activity-feed": "^0.9.26",
    "react-dom": "^16.12.0",
    "react-scripts": "3.4.0"
  }
```

Am I missing something on this setup?

**Comment by jaapbakker88** [comment]
Do you see the messages in the console saying "now listening for realtime...."

Can you add the `userId` prop to the FlatFeed component.

**Comment by rkulinski** [comment]
I'm having the same issue. Passing `userId` didn't help.

**Comment by jaapbakker88** [comment]
yeah it's not live updating for removeActivity, only on new activities(which wil show the "you have X new activities" banner). For others you wil have to refresh, which is intended behaviour.

**Comment by rkulinski** [comment]
@jaapbakker88 how should I know "in browser" when server deleted some activity (let's say another user deleted it)?

**Comment by augustosamame** [comment]
We also have this issue. After spending weeks setting up a full fledged integration with getstream.io, we find that the feed is not updated for removeActivity, which renders our whole integration useless.
Not sure why this would be "intended behaviour", but since it is, there needs to be a way for the component to detect the change in the feed data and trigger the refresh.

**Comment by augustosamame** [comment]
Since getstream people seem to be unresponsive, here's what we did to solve this issue.
Whenever an activity is deleted, there is actually a websocket communication sent:

![image](https://user-images.githubusercontent.com/16038063/119008659-fcd45800-b957-11eb-849b-09f7365bbf44.png)

You can listen for this data stream and reload the component to show the updated feed.

Hope this helps someone that has the same issue we did.
Shame on getstream for their non-existing support even for paid accounts.

**Comment by Rykuno** [comment]
I'm running into this issue as well. When a user's page isnt cached within NextJS it listens for new changes, but navigating back to a subsequent user's feed after caching it does not trigger. I was hoping there would be a manual way to go about this.

**Comment by vuthanhtrung0504** [comment]
Are there any update on this yet? I'm also have this isssue
